src/main/java/lotto/domain/LottoGame.java

```diff
@@ -0,0 +1,59 @@
+package lotto.domain;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class LottoGame {
+
+    private static final int ZERO = 0;
+
+    private final StringBuilder stringBuilder = new StringBuilder();
+    private final Map<Ranking, Integer> totalResult = new HashMap<>();
+
+    public LottoGame() {
+        initializeTotalResult();
+    }
+
+    private void initializeTotalResult() {
+        totalResult.put(Ranking.FIRST, ZERO);
+        totalResult.put(Ranking.SECOND, ZERO);
+        totalResult.put(Ranking.THIRD, ZERO);
+        totalResult.put(Ranking.FOURTH, ZERO);
+        totalResult.put(Ranking.FIFTH, ZERO);
```

Comment:
만약 7등, 8등 복권이 생긴다면 복권이 새로 생길 때마다 이 메서드는 한 줄씩 늘어날거에요! 🥲
아래와 같이 변경해보면 어떨까요? 😃 
```suggestion
        Arrays.stream(Ranking.values())
                .forEach(ranking -> totalResult.put(ranking, ZERO));
```

Reply:
헉! 너무 하드코딩 되어 있었네요! 감사합니다~